tests/unit_tests/test_cylindrical_mesh.py

```diff
@@ -98,8 +98,99 @@ def test_offset_mesh(model, estimator, origin):
         centroids = mesh.centroids
         for ijk in mesh.indices:
             i, j, k = np.array(ijk) - 1
-            print(centroids[:, i, j, k])
             if model.geometry.find(centroids[:, i, j, k]):
                 mean[i, j, k] == 0.0
             else:
                 mean[i, j, k] != 0.0
+
+@pytest.fixture()
+def void_coincident_geom_model():
+    """A model with many geometric boundaries coincident with mesh boundaries
+       across many scales
+    """
+    openmc.reset_auto_ids()
+    model = openmc.model.Model()
+
+    model.materials = openmc.Materials()
+    radii = [0.1,1, 5, 50, 100, 150, 250]
+    spheres = [openmc.Sphere(r=ri) for ri in radii]
```

Comment:
Should this be cylinders instead of spheres?

Reply:
Mhmm. Just fixed. Thanks!